packages/@vue/cli-plugin-typescript/generator/template-vue3/src/shims-vue.d.ts

```diff
@@ -1,5 +1,5 @@
 declare module '*.vue' {
   import type { DefineComponent } from 'vue'
-  const component: DefineComponent
+  const component: DefineComponent<any, any, any>
```

Comment:
`DefineComponent<{}, {}, any>`

maybe?

Reply:
👍 Yes, now that VTU beta.7 is out, this is enough.